authors/apps/articles/filter.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.generics import ListAPIView
+from django_filters.rest_framework import DjangoFilterBackend
```

Comment:
DjangoFilterBackend should be available as filters.DjangoFilterBackend from the import on line 3